src/app/test/test.component.html

```diff
@@ -9,33 +9,15 @@
       <button data-cy-test="refresh" class="btn btn-info" title="ReloadTestReports"
               (click)="loadData()"><i class="fa fa-refresh"></i></button>
       <button class="btn btn-info" title="Reset" (click)="resetRunner()">Reset</button>
-      <button class="btn btn-info" title="Options" (click)="openSettingsModal()"><i class="fa fa-cog"></i></button>
-      <button
-        data-cy-test="selectAll"
-        class="btn btn-info"
-        title="Select All"
-        (click)="setCheckedForAllReports(true)"
-      >Select All
-      </button>
-      <button data-cy-test="deselectAll"
-              class="btn btn-info"
-              title="Deselect All"
-              (click)="setCheckedForAllReports(false)">
-        Deselect All
-      </button>
-      <button data-cy-test="deleteSelected" class="btn btn-info" title="Delete Selected"
-              (click)="openDeleteModal()"><i class="fa fa-trash"></i></button>
-      <button class="btn btn-info" title="Clone report"
-              (click)="openCloneModal()">Clone
-      </button>
-      <button data-cy-test="copySelected" class="btn btn-info" title="Copy Selected"
-              (click)="copySelected()"><i class="fa fa-copy"></i></button>
-      <button data-cy-test="downloadBinary" class="btn btn-info" title="Download Binary"
-              (click)="downloadSelected()"><i class="fa fa-download"></i></button>
+      <button class="btn btn-info" title="Options" (click)="testSettingsModal.open()"><i class="fa fa-cog"></i></button>
+      <button data-cy-test="deleteSelected" class="btn btn-info" title="Delete Selected" (click)="openDeleteModal()"><i class="fa fa-trash"></i></button>
+      <button class="btn btn-info" title="Clone report" (click)="openCloneModal()">Clone</button>
+      <button data-cy-test="copySelected" class="btn btn-info" title="Copy Selected" (click)="copySelected()"><i class="fa fa-copy"></i></button>
+      <button data-cy-test="downloadBinary" class="btn btn-info" title="Download Binary" (click)="downloadSelected()"><i class="fa fa-download"></i></button>
       <button data-cy-test="upload" type="button" class="btn btn-info" title="Upload"
               (click)="uploadFileTest.click()">
         <i class="fa fa-upload"></i>
-        <input #uploadFileTest data-cy-test="uploadFile" type="file" id="uploadFileTest"
+        <input #uploadFileTest data-cy-test="uploadFile" type="file" [hidden]="true"
```

Comment:
Why is this input always hidden?

Reply:
It's for uploading files, but the default input element cannot be styled very nicely.